cypress/support/step_definitions/fireboltCalls.js

```diff
@@ -325,9 +331,11 @@ Given(/1st party app registers for the '(.+)' event using the '(.+)' API$/, asyn
             if (UTILS.getEnvVariable(CONSTANTS.CERTIFICATION) == true) {
               assert(false, `${CONSTANTS.PLATFORM_NOT_SUPPORT_LOG}: ${method}`);
             } else {
-              cy.log(`NotSupported: ${CONSTANTS.PLATFORM_NOT_SUPPORT_LOG}: ${method}`).then(() => {
-                throw new Error(CONSTANTS.STEP_IMPLEMENTATION_MISSING);
-              });
+              fireLog
```

Comment:
Why is firelog and assert in new lines? If this was not done due to linting, a single line is more meaningful

Reply:
This was from linting.